svc-lighthouse-api/src/integrationTest/resources/application-integrationTest.properties

```diff
@@ -1,3 +1,8 @@
+# Many of these aren't needed since application.properties should already set them to the same values
+# TODO: run the tests with `@ActiveProfiles("integrationTest")` and remove unnecessary settings below
+#   See application-test.properties for an example.
+#   May even be able to use `@ActiveProfiles({"test","integrationTest"})` to remove more settings
```

Comment:
@wlin2019 Can you address this TODO? application.properties should be loaded, along with this file, so you don't need to set the same properties again.

Reply:
Ok. I'll create a Jira ticket for it. By the way, I moved those tests to integrationTest from test as they were not unit tests. However, we have a lot of "integration" tests in test, and threw out the rules of UNIT test. Do we still need this integrationTest?